commands/commands.go

```diff
@@ -218,6 +222,28 @@ func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclu
 	return includePaths, excludePaths
 }
 
+// help is used for 'git-lfs help <command>'
+func help(cmd *cobra.Command, args []string) {
+	if txt, ok := ManPages[cmd.Use]; ok {
+		fmt.Fprintf(os.Stderr, txt)
+	} else {
+		fmt.Fprintf(os.Stderr, "Sorry, no help text found\n")
+	}
```

Comment:
I don't think `cmd.Use` is what you want. I updated the error message like this:

```go
fmt.Fprintf(os.Stderr, "Sorry, no help text found for %q.\n", cmd.Use)
```

And got: "help [command]". I think `args[0]` is what you want.